### PR TITLE
Create index

### PR DESCRIPTION
**Pull request by hjohnson0586**
Added api endpoint, index, and related tests.

### CODE REVIEW

File: app/javascript/react/components/GameIndex.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    return (
+      <div>
+      <h1>Games</h1>
+      <h3>{gameIndexHTML}</h3>
+      </div>
+    )
+  }
+}
+
+export default GameIndex;
```
Comment (bkincart) [inline app/javascript/react/components/GameIndex.js:45]: Please check and resolve your formatting in this file! It needs major indentation and removal of empty lines

File: app/javascript/react/components/app.js [hunk 2]
```diff
@@ -4,4 +4,4 @@ export const App = (props) => {
   return (<h1>Make It So React</h1>)
 }
 
-export default App
\ No newline at end of file
+export default App
```
Comment (bkincart) [inline app/javascript/react/components/app.js]: You can remove this file since you're not using it!

File: app/views/games/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (bkincart) [inline app/views/games/index.html.erb]: Remove empty line

File: app/views/homes/index.html.erb [hunk 4]
```diff
@@ -1,4 +1,3 @@
 <p>
-  Modify me in `app/views/homes/index.html.erb` or change the root directive in
-  `config/routes.rb`
+
 </p>
```
Comment (bkincart) [inline app/views/homes/index.html.erb]: If this erb file is no longer being used I'd remove this as well!

File: spec/controllers/api/v1/games_controller_spec.rb [hunk 5]
```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::GamesController, type: :controller do
+  before(:each) do
+    Game.create(name: "Overwatch", description: "copying the video", year: '11', esrb: "M", promo_image_url: "image")
+    Game.create(name: 'Call of Duty: Black Ops 4', description: 'COD Black Ops 4 is a shooting game', promo_image_url: 'https://orig00.deviantart.net/145b/f/2018/138/a/6/call_of_duty_black_ops_4___icon_by_blagoicons-dcbu90s.png',
+    year: '2018', esrb: 'M')
```
Comment (bkincart) [inline spec/controllers/api/v1/games_controller_spec.rb:7]: I love that you have a factory!! You should use it here!

**Review by bkincart — DISMISSED** [review]

File: app/views/games/index.html.erb [hunk 6]
```diff
@@ -0,0 +1 @@
+<div id="app"></div>
```
Comment (CassandraNewell) [inline app/views/games/index.html.erb:1]: fyi, a shorter version is 
```suggestion
<div id="app" />
```
(self-closing tag)

**Review by CassandraNewell — APPROVED** [review]

**Pull request merged**